tools/build-updatedhelp.ps1

```diff
@@ -0,0 +1,141 @@
+<#
+    Usage:
+
+    1. Clone the docs respository
+       cd C:\temp
+       git clone https://github.com/MicrosoftDocs/PowerShell-Docs.git
+
+    2. Run build-updatedhelp.ps1 for the version folder you want to build
+
+      .\build-updatedhelp.ps1 -sourceFolder C:\temp\PowerShell-Docs\reference\5.1 -Verbose
+
+    3. Run Update-Help  to install the newly built help
+
+       Update-Help -SourcePath c:\temp\updatablehelp\5.1 -Recurse -Force
+#>
+
+
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateScript({ Test-Path $_ })]
+    [string]$sourceFolder
+)
+
+$tempDir = [System.IO.Path]::GetTempPath()
+
+# Pandoc source URL
+$panDocVersion = "2.7.3"
+$pandocSourceURL = "https://github.com/jgm/pandoc/releases/download/$panDocVersion/pandoc-$panDocVersion-windows-x86_64.zip"
+
+$docToolsPath = New-Item (Join-Path $tempDir "doctools") -ItemType Directory -Force
+$pandocZipPath = Join-Path $docToolsPath "pandoc-$panDocVersion-windows-x86_64.zip"
+Write-Verbose "Downloading Pandoc..."
+Invoke-WebRequest -Uri $pandocSourceURL -OutFile $pandocZipPath
```

Comment:
Not a huge deal, but on WinPS5.1 progress slows down invoke-webrequest and expand-archive quite a bit.  This isn't an issue in PSCore6.  Don't know how big the zip is so it may not be a big issue even on 5.1.  Otherwise, you may want to special case 5.1 by changing progresspreference to silentlycontinue and then changing it back to the original value.

Reply:
Fixed